taca/utils/bioinfo_tab.py

```diff
@@ -18,6 +19,12 @@ def setupServer(conf):
     url="http://{0}:{1}@{2}:{3}".format(db_conf['username'], db_conf['password'], db_conf['url'], db_conf['port'])
     return couchdb.Server(url)
 
+#Constructor for a search tree
+class Tree(defaultdict):
+    def __init__(self, value=None):
+        super(Tree, self).__init__(Tree)
+        self.value = value
+
```

Comment:
yeah, but you need to declare a new dict for each layer. Tree allows you to avoid that.